src/app/modules/activities-management/store/activity-management.actions.ts

```diff
@@ -9,6 +9,9 @@ export enum ActivityManagementActionTypes {
   CREATE_ACTIVITY = '[ActivityManagement] CREATE_ACTIVITY',
   CREATE_ACTIVITY_SUCCESS = '[ActivityManagement] CREATE_ACTIVITY_SUCCESS',
   CREATE_ACTIVITY_FAIL = '[ActivityManagement] CREATE_ACTIVITY_FAIL',
+  DELETE_ACTIVITY = '[ActivityManagement] Delete Activity',
+  DELETE_ACTIVITY_SUCCESS = '[ActivityManagement] Delete Activity Success',
```

Comment:
Please, let's keep the same format in all actions (capital letters)

Reply:
done